_opcodes/op453-iwd2.html

```diff
@@ -10,4 +10,10 @@
 iwd2: 1
 pst: 0
 ---
-Disables the usage of Expertise, detects the activation/revocation of Power Attack, prints appropriate messages. This effect is used internally.
+Disables the usage of Expertise, detects the activation/revocation of Power Attack, prints appropriate messages.
+
+{% capture note %}
+This effect is used internally.
```

Comment:
The note says just that the effect is used by the engine, nothing more. The engine also uses the damage opcode and others, but they don't have this note. It needs more research whether these modal feat effects are any special at all. That's why it's pointless to stress the fact.